### PR TITLE
Modify an error description

### PR DESCRIPTION
**Pull request by lyush**

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://fefe5ee3a101ddc92d82-cf0186b253f52b7e10aa6aa299175de7.r11.cf5.rackcdn.com/ to review your changes.

**Comment by lyush** [comment]
thanks, it's ok.

File: guides/openstack.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ In the constructor note that
   * This ones pretty self explanatory, we're using the OpenStack Nova provider in jclouds
 * `String identity = "demo:demo"; // tenantName:userName`
   * Here we use the tenant name and user name with a colon between them instead of just a user name
-* `String password = "devstack";`
+* `String credential = "devstack";`
```
Comment (demobox) [inline guides/openstack.md]: Looks good to me - thanks, @lyush!

**Comment by nacx** [comment]
Merged and updated the site! Thanks!

**Pull request closed without merging**